### ISSUE TITLE
Inheritance security rules violated while overriding member:

### ISSUE THREAD

**Issue by mohdwaseem** [quartznet/quartznet#265]
![quartz](https://cloud.githubusercontent.com/assets/3831663/7610087/9991141c-f997-11e4-9d72-82d69cbee686.PNG)
Inheritance security rules violated while overriding member: 'Quartz.Core.QuartzScheduler.InitializeLifetimeService()'. Security accessibility of the overriding method must match the security accessibility of the method being overriden.

I got this error i hosted my application on godaddy shared hosting please help me to fix this problem. Or any other alternate for scheduling the task

**Comment by lahma** [comment]
duplicate of #258

**Comment by cebeser** [comment]
What is the solution to this problem? Anyone knows?
